client/src/components/HistoryPage/index.tsx

```diff
@@ -13,7 +13,17 @@ const ActiveBookings = observer(() => {
   const handleDelete = () => {
     //TODO: sjekk om det er event admin som sletter reservasjon gjort av noen andre
     // kanskje event admin skal få se navn under sete nummer på Reservasjoner/History page
-    historyStore.deleteBooking(historyStore.bookingIdToDelete);
+
+    const bookingToDelete = historyStore.myActiveBookings.find(historyBooking => historyBooking.id === historyStore.historyBookingIdToDelete);
```

Comment:
Vi sender inn booking.id til <BookingItem /> via onClick på linje 47. Da oppdaterer storen bookingToDelete. Altså storen har call på hvilket BookingItem som er valgt. Tenker derfor at man ikke trenger å hente id'en ut for å så sende den inn igjen på linje 21 og 24. Tanker.